### DISCUSSION TITLE
Support AreaShop, you have AdvancedRegionMarket

### DISCUSSION THREAD

**Discussion by TomLewis** [QuickShop-Community/QuickShop-Hikari#1737] [Ideas]
Please support AreaShop as well as AdvancedRegionMarket.

Updated Version https://github.com/md5sha256/AreaShop
Legacy version that's no longer supported for more info about the plugin https://www.spigotmc.org/resources/areashop.2991/

Very popular, we've used it for many years on piratemc.com and have an established player marketplace https://map.piratemc.com/survival#PirateCraft_1;surface;-1926,64,472;5

I plan to move to QuickShop from SignShop, but this seems to the main issue QuickShop, that it has no Support for areashop yet.

Thank you.

**Comment by creatorfromhell** [comment]
> Please support AreaShop as well as AdvancedRegionMarket.
> 
> Updated Version https://github.com/md5sha256/AreaShop Legacy version that's no longer supported for more info about the plugin https://www.spigotmc.org/resources/areashop.2991/
> 
> Very popular, we've used it for many years on piratemc.com and have an established player marketplace https://map.piratemc.com/survival#PirateCraft_1;surface;-1926,64,472;5
> 
> I plan to move to QuickShop from SignShop, but this seems to the main issue QuickShop, that it has no Support for areashop yet.
> 
> Thank you.

Could you specify what functionality doesn't currently work/what would be needed from the addon/compat?

**Comment by TomLewis** [comment]
When you unrent a shop the hologram of the item still exists, it thinks the shops still there.
It works the same way as AdvancedRegionMarket, where you rent a shop and when it unrents a schematic is pasted over where the shop was to clear it for the next person to rent.

Sorry first time ever using this plugin.
This is the shop as I set it up as a rented shop
![2024-08-25_14 07 11](https://github.com/user-attachments/assets/ad42968b-1d52-4744-bddf-683e9ed234cc)

Then when I unrent the shop it wipes it and resets it to the default shop schematic, leaving this hologram and the item
![2024-08-25_14 07 59](https://github.com/user-attachments/assets/6970beac-5b54-45d4-8552-ab2da5d6b30d)

but if I place a chest there and right click it, it thinks the shops still there? but the hologram is where it was on the floor
![2024-08-25_14 10 27](https://github.com/user-attachments/assets/3cea831e-5baf-402c-aa23-ffc020767092)

Luckily /qs remove works to remove it.

AreaShop uses WorldGuard if that helps, there are events for when a shop unrents to hook into, so anything in that shop region when unrented needs to be removed as a shop.

I cant think of any other events that need to be listened for? maybe shop delete? that's another event for admins to completely delete a shop.

**Comment by TomLewis** [comment]
Bumping

**Reply by creatorfromhell** [reply]
Follow: https://github.com/QuickShop-Community/QuickShop-Hikari/issues/1756 for updates on this.

**Reply by creatorfromhell** [reply]
relooking at this, the maven on the wiki linked on their README isn't active anymore...
